src/main/java/org/owasp/validator/html/model/Attribute.java

```diff
@@ -143,6 +143,35 @@ public String matcherRegEx(boolean hasNext){
             }
         }
         return regExp.toString();
+    }
+
+    public static String mergeRelValuesInAnchor(boolean addNofollow, boolean addNoopenerAndNoreferrer, String currentRelValue) {
```

Comment:
Please add JavaDoc explaining this method signature.

Reply:
Done